content/applications/websites/livechat/chatbots.rst

```diff
@@ -0,0 +1,214 @@
+         Question steps should only be used when visitors are provided options to select.
+
+   .. tab:: Email
+
+      This step prompts visitors to provide their email address, which will be stored and can be
+      used by team members later to follow up with additional information.
+
+      The only accepted inputs for this step-type are email addresses that are in a valid format.
+      If a visitor attempts to enter anything other than a valid email address, the chatbot will
+      respond with a message stating it does not recognize the information submitted.
+
+      .. image:: chatbot/chatbot-invalid-email.png
+         :align: center
+         :alt: View of a chatbot responding to an invalid email.
+
+   .. tab:: Phone
+
+      Similar to email, this step type prompts the visitor to enter their phone number, which can
+      be used at a later time to follow up with additional information, or to schedule demos, etc.
+
+   .. tab:: Forward to Operator
+
+      This step forwards the conversation on to an active live chat operator, so that they can
+      continue assisting the visitor. As the conversation transcript is passed on to the operator
+      as well, they will be able to pick up where the chatbot left off. This not only saves time,
+      for all parties involved, but it can help qualify conversations before they reach live human
+      operators.
+
+      .. note::
+        If no active operator is available on the channel, the chatbot will continue the
+        conversation with the visitor. Therefore, additional steps should be added after this one
+        to ensure that there is no abrupt end to the conversation.
+
+        .. image:: chatbot/chatbot-no-operator.png
+           :align: center
+           :alt: View of a chatbot follow up messages when no live chat operator is available.
+
+   .. tab:: Free Input/Multi-Line
+
+      The free input step allows visitors to respond to questions without providing pre-written
+      responses. Information provided in these responses will be stored in the chat transcripts.
+      Choose between :guilabel:`Free Input` and :guilabel:`Free Input (Multi-Line)` depending on
+      the type and amount of information being requested from the visitor.
+
+   .. tab:: Create Lead
+
+      This step will create a lead in the :guilabel:`CRM` application, and assign it to a
+      specific team (as identified in the appropriate field.)
+
+   .. tab:: Create Ticket
+
+      This step will create a ticket in the :guilabel:`Helpdesk` application, and assign it to a
+      specific team (as identified in the appropriate field.)
+
+Only if
+~~~~~~~
+
+Chatbot scripts operate on an if/then basis, which means that the next question presented to the
+visitor is determined by the answer they provide to the previous question. To continue the
+progression of the conversation, the form for a new step contains a field labeled *Only If*. This
+field is where the progression of questions is defined.
```

Comment:
```suggestion
Chatbot scripts operate on an if/then basis, which means that the next question presented to the
visitor is determined by the answer they provide to the previous question.

To continue the progression of the conversation, the form for a new step contains a field labeled *Only If*. This
field is where the progression of questions is defined.
```

Reply:
Looks wonky here but my suggestion was to break up into 2 paragraphs at "question..." and "...To"